.github/workflows/ci.yml

```diff
@@ -381,7 +381,7 @@ jobs:
     runs-on: windows-2019
     timeout-minutes: 30
     env:
-        VFLAGS: -cc msvc
+        VFLAGS: -cc msvc -cflags -Werror
```

Comment:
I think msvc uses `/WX` instead of `-Werror`

Reply:
https://docs.microsoft.com/en-us/cpp/build/reference/compiler-option-warning-level?view=msvc-160

Reply:
Also keep in mind that adding it in VFLAGS, will use that for all the subsequent compilations, including tests and compiling the examples.

Reply:
Oh, right. Thank you :slightly_smiling_face:

Reply:
@spytheman what should I do instead of "VFLAGS" for msvc?

Reply:
Just pass it on the cli with -cflags /WX during the v self compilation:
`v -cflags /WX self`

Reply:
or  `v -o v2.exe /WX self` , if you want to produce another executable (v self is smart enough to know to rename the file at the end, so that it works on windows too)

Reply:
Thank you :slightly_smiling_face: I was not aware that "-cflags" had to come before "self"

Reply:
Do you intend to make other changes, or is it good to merge?

Reply:
I think it is ready to be merged :slightly_smiling_face: